spec/policies/course_policy_spec.rb

```diff
@@ -0,0 +1,13 @@
+describe CoursePolicy do
+  let(:context) { { user: user } }
+  let(:user) { create :student }
+  describe_rule :manage? do
```

Comment:
`manage?` is no longer defined for this policy (it's inherited) so you can remove this test here